### PR TITLE
UI upgraded: only showing for ERC20-->BEP2

### PR DESCRIPTION
**Pull request by Renewwen**
BEP2 ---> ERC20
image
ERC20 ---> BEP2
image

### CODE REVIEW

File: bnbridge/src/components/instructions/instructions.jsx [hunk 1]
```diff
@@ -68,6 +68,11 @@ const styles = theme => ({
 });
 
 class Instructions extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+  }
+
```
Comment (denniswon) [inline bnbridge/src/components/instructions/instructions.jsx]: unncessary: lets remove

File: bnbridge/src/components/instructions/instructions.jsx [hunk 2]
```diff
@@ -118,7 +123,7 @@ class Instructions extends Component {
     })
   };
 
-  render() {
+  renderBEP2_ERC20() {
```
Comment (denniswon) [inline bnbridge/src/components/instructions/instructions.jsx]: breaking function naming convention: change to renderBep2ToErc20 (use lowerCamelCase)

File: bnbridge/src/components/instructions/instructions.jsx [hunk 3]
```diff
@@ -132,7 +137,28 @@ class Instructions extends Component {
...
             <Typography className={classes.header} style={{ marginTop: '10px' }}>With bnbridge you can:</Typography>
             <li><Typography className={classes.action}>Swap between ERC20 and BEP2 tokens</Typography></li>
-            
+          </div>
+        </Grid>
+      </Grid>
+    )
+  };
+
+  renderERC20_BEP2() {
```
Comment (denniswon) [inline bnbridge/src/components/instructions/instructions.jsx]: ditto: use lowerCamelCase

File: bnbridge/src/components/instructions/instructions.jsx [hunk 4]
```diff
@@ -147,6 +173,20 @@ class Instructions extends Component {
...
+  render() {
+    const {
+      classes,
+      swapDir
+    } = this.props;
+
+    return (
+      <Grid container className={ classes.root }>
+          { swapDir === 1 && this.renderBEP2_ERC20() }
+          { swapDir === 2 && this.renderERC20_BEP2() }
```
Comment (denniswon) [inline bnbridge/src/components/instructions/instructions.jsx]: define const or enum for swapDir instead of using 1, 2 hardcoded. makes code readable

**Review by denniswon — CHANGES_REQUESTED** [review]
Nit change requests.

**Comment by denniswon** [comment]
Also, can you make the screenshot size for the memo smaller and then centered horizontally within the component? (like 80%) just seems like the screenshot texts are as big as the websites own texts, and not clear which one is which.

**Comment by Renewwen** [comment]
After changed: (the size of screenshot to 80%)
image

File: bnbridge/src/components/instructions/instructions.jsx [hunk 5]
```diff
@@ -181,8 +176,8 @@ class Instructions extends Component {
 
     return (
       <Grid container className={ classes.root }>
-          { swapDir === 1 && this.renderBEP2_ERC20() }
-          { swapDir === 2 && this.renderERC20_BEP2() }
+          { swapDir === 'Bep2ToErc20' && this.renderBep2ToErc20() }
```
Comment (denniswon) [inline bnbridge/src/components/instructions/instructions.jsx:179]: you are still hardcoding not constants or enum

Comment (denniswon) [reply bnbridge/src/components/instructions/instructions.jsx:179]: I will fix this.

**Review by denniswon — APPROVED** [review]

**Pull request merged**